### PR TITLE
missing `dokku` on `shell command`

### PR DESCRIPTION
**Pull request by flashios09**

### CODE REVIEW

File: docs/deployment/user-management.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ b7:76:27:4f:30:90:21:ae:d4:1e:70:20:35:3f:06:d6
 Admin users and root can also add keys remotely:
 
 ```shell
-cat ~/.ssh/id_rsa.pub | ssh dokku@dokku.me ssh-keys:add KEY_NAME
+cat ~/.ssh/id_rsa.pub | ssh dokku@dokku.me dokku ssh-keys:add KEY_NAME
```
Comment (josegonzalez) [inline docs/deployment/user-management.md]: This shouldn't be necessary as `sshcommand` will autoprefix anything sent to the dokku user via ssh with the `dokku` bin.

Comment (flashios09) [inline docs/deployment/user-management.md]: unfortunately this doesn't work for me
```fish
fish: Unknown command 'ssh-keys:add flashios09-iMac'
fish: ssh-keys:add flashios09-iMac
```

Comment (josegonzalez) [inline docs/deployment/user-management.md]: Does it work with the prefix? What version of dokku are you using?

Comment (flashios09) [inline docs/deployment/user-management.md]: this command works for me:
```
cat ~/.ssh/id_rsa.pub | ssh dokku@dokku.me dokku ssh-keys:add KEY_NAME
```
I'm using dokku 0.8.2

Comment (josegonzalez) [inline docs/deployment/user-management.md]: @michaelshobbs this shouldn't be possible, right?

Comment (michaelshobbs) [inline docs/deployment/user-management.md]: depends on what his `authorized_keys` file already looks like

Comment (josegonzalez) [inline docs/deployment/user-management.md]: @flashios09 whats in your `authorized_keys` file?

**Comment by josegonzalez** [comment]
This was because the user for ssh wasn't `dokku`, but their root user.

**Pull request closed without merging**